polyglot/piranha/src/lib.rs

```diff
@@ -93,11 +96,30 @@ pub fn execute_piranha(
   }
 
   // flag_cleaner.relevant_files
-  flag_cleaner
+  let summaries = flag_cleaner
     .get_updated_files()
     .iter()
     .map(PiranhaOutputSummary::new)
-    .collect_vec()
+    .collect_vec();
+  log_the_summary(&summaries);
+  return summaries;
+}
+
+fn log_the_summary(summaries: &Vec<PiranhaOutputSummary>) {
+    let mut total_number_of_matches : usize = summaries.iter().map(|x|x.matches().len()).sum();
+    let mut total_number_of_rewrites : usize = summaries.iter().map(|x|x.rewrites().len()).sum();
+    for summary in summaries {
+      let number_of_rewrites =&summary.rewrites().len();
+      let number_of_matches = &summary.matches().len();
+      info!("File : {:?}", &summary.path());
+      info!("{}", format!("# Rewrites : {number_of_rewrites}"));
```

Comment:
Isn't info already taking a format string? Why the double templating? (Also, shouldn't we be indenting per-file results?)

Reply:
`# Rewrites : {number_of_rewrites}"` being able to lookup a variable `number_of_rewrites` in the context to format the string is only possible using the `format!` macro.

Reply:
At some places I am doing double formatting because I add color to the logs using the `format!` macro.